### PR TITLE
add pr workflow

### PR DESCRIPTION
**Pull request by okigan**
- [ x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: .github/workflows/check_pr.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Generate formulae.brew.sh data.
+
+on: [push]
+
+jobs:
+  generate:
+    if: github.repository == 'Homebrew/homebrew-core'
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@master
```
Comment (chenrui333) [inline .github/workflows/check_pr.yml]: might be good to use release tag instead of master.

**Comment by SMillerDev** [comment]
What are you actually trying to do? Right now it doesn't do anything useful.

**Comment by okigan** [comment]
@SMillerDev this is accidental PR, but since it got your attention how about moving the checklist items from checklist into github actions? (so steps 3-5 are performed automatically)

**Comment by SMillerDev** [comment]
Because that is too late. We don't want people making 25 updates because they didn't run the command locally. As for checking if people did actually run the commands, that already happens on Jenkins.

**Pull request closed without merging**